zerver/openapi/zulip.yaml

```diff
@@ -11849,21 +11843,19 @@ paths:
                           huddles:
                             type: array
                             description: |
-                              An array of dictionaries where each entry contains details
-                              of unread group direct messages with a specific group of users.
+                              An array of objects where each object contains details of unread
+                              group direct messages with a specific group of users.
                             items:
                               type: object
-                              description: |
-                                Object containing the details of unread group direct messages
-                                with a specific group of users.
                               additionalProperties: false
                               properties:
                                 user_ids_string:
                                   type: string
                                   description: |
                                     A string containing the IDs of all users in the group
-                                    direct message conversation separated by commas; for
-                                    example: `"1,2,3"`.
+                                    direct message conversation, including the current user,
+                                    separated by commas and sorted numerically; for example:
+                                    `"1,2,3"`.
```

Comment:
🎉

Reply:
This is helpful! As @andersk gently reminded me on [CZO](https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/register.3A.20.60unread_msgs.2Ehuddles.5B.5D.2Euser_ids_string.60/near/1641025), this change is more substantive than a clarification/cleanup, since it changes the set of guarantees that are made. So I would suggest either highlighting it in the commit message or separating it out into a different commit.

Reply:
Pulled this out to a separate, initial commit.